client/src/App.js

```diff
@@ -26,7 +26,7 @@ const App = () => (
 			<Route path="/bidder-dashboard" element={<BidderDashboard />} />
 			<Route path="/list-tenders" element={<TendersList />} />
 			<Route path="/list-tenders/page/:pageNumber" element={<TendersList />} />
-			<Route path="/login" element={<LogInForm />} />
+			<Route path="/sign-in" element={<LogInForm />} />
```

Comment:
Delete this route. Our sign-in form is already in `/`

Reply:
Alright